arcs/crowdflower.py

```diff
@@ -46,11 +47,13 @@ def create_job_from_copy(api_key=None, job_id=None):
         job_id (int): The unique job identifier of the job to copy
 
     Returns:
-        A new Arcs Job
+        A new Job
     """
-    job_id = job_id or 844991
+    job_id = job_id or 911946
```

Comment:
Not at all. I thought it would be useful, but ultimately, it was a terrible idea, because it makes it easy to forget to update this default, which you want to do to get the most current set of test data for a task.